pkg/util/service/service.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
+
+const (
+	// LoadBalancerProtectionFinalizer is the finalizer added to load balancer services to ensure
+	// that after successful creation the cloud resources are removed before deleting the service.
+	LoadBalancerProtectionFinalizer = "kubernetes.io/load-balancer-service-protection"
```

Comment:
Thoughts on using `networking.kubernetes.io/load-balancer-protection` instead?

Reply:
or `networking.kubernetes.io/load-balancer-finalizer`

Reply:
@andrewsykim I picked this name so there was consistency with the PV protection.
https://github.com/kubernetes/kubernetes/blob/b11211ed8cfbf5b613c9dfb20affa6627c5e7ac7/pkg/volume/util/finalizer.go#L24